src/main/java/it/gov/pagopa/payhub/activities/dao/OrganizationInstallmentTypeDao.java

```diff
@@ -0,0 +1,10 @@
+package it.gov.pagopa.payhub.activities.dao;
+
+import it.gov.pagopa.payhub.activities.dto.OrganizationInstallmentTypeDTO;
+
+import java.util.List;
+
+public interface OrganizationInstallmentTypeDao {
+
+    List<OrganizationInstallmentTypeDTO> getByMygovEnteIdAndOperatoreUsername(Long mygovEnteId, String operatoreUsername);
```

Comment:
let's clean the name and javadoc, in order to support who will implement it
```suggestion
    /** It will return DebtPositionTypeOrg entities authorized to the input operato */
    List<DebtPositionTypeOrgDTO> getAuthorizedDebtPositionTypeOrgs(Long orgId, String operatorUsername);
```